### PR TITLE
Devicemanagement login

### PR DESCRIPTION
**Pull request by reshma-qburst**
Changes in this branch:

1) Routing implementation.
2) Bootstrap css for styling.
3) Login features:
- UI
- Validations
- Sample JSON with an admin and a normal user and validated against the entered username and password.
- Redirection to admin/ user dashboard based on the role of logged in user.
4) Authguard Implementation
- Restricted logged in users from accessing the "login" page.
- Restricted non logged in users from accessing the inner pages (which require login).
- Restricted normal user accessing admin privileged pages and vice versa.
5) Logout functionality.
6) Dashboard & Account settings UI changes.
7) Created Header, Footer sections as separate components.

### CODE REVIEW

File: src/app/accountsettings/accountsettings.component.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// Account settings component.
+
+import { Component, OnInit } from "@angular/core";
```
Comment (nishintn) [inline src/app/accountsettings/accountsettings.component.ts]: what is OnInit?

Comment (reshma-qburst) [inline src/app/accountsettings/accountsettings.component.ts]: OnInit class is one of Angular's lifecycle hook. It is initiated only once when the directive is instantiated. It has ngOnInit() method where in required code is written.

File: src/app/guards/login.guard.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// Prevent loggedin users from accessing login route.
```
Comment (nishintn) [inline src/app/guards/login.guard.ts]: @rekharv ,
Why do we need separate guard for log in?

Comment (rekharv) [inline src/app/guards/login.guard.ts]: This was for preventing the logged in users accessing the "login" page. The other guard service we used was basically for inner pages. So for better understanding we separated the guards.

**Pull request merged**